javascripts/releases.js

```diff
@@ -0,0 +1,78 @@
+var ev3devRepoReleaseCacheKey = 'ev3dev-repo-release-cache';
+var releaseCacheTimeMillis = 1 * 24 * 60 * 60 * 1000;
+
+var releasePlatformRegexes = {
+    ev3: "ev3-[\\w\\d-]*\\.img\\.xz",
+    rpi: "rpi-[\\w\\d-]*\\.img\\.xz",
+    rpi2: "rpi2-[\\w\\d-]*\\.img\\.xz",
+    evb: "evb-[\\w\\d-]*\\.img\\.xz",
+}
+
+function supportsHtml5Storage() {
+    try {
+        return 'localStorage' in window && window['localStorage'] !== null;
```

Comment:
It should persist until the user manually does something to clear it; I suspect that most browsers will clear it on forced server refresh as you mentioned. I mainly did this because GitHub rate limits unauthenticated users; the rate limit is 60/hour, so obviously I should be able to increase the cache time to something more like 30 minutes or 10 minutes. I'd like to add caching for the GitHub user cards as well to make sure that no user starts seeing broken pages.